appinventor/buildserver/src/com/google/appinventor/buildserver/Compiler.java

```diff
@@ -319,7 +320,7 @@ private boolean writeAndroidManifest(File manifestFile, Set<String> permissionsN
 
     // TODO(user): Use com.google.common.xml.XmlWriter
     try {
-      BufferedWriter out = new BufferedWriter(new FileWriter(manifestFile));
+      BufferedWriter out = new BufferedWriter(new OutputStreamWriter(new FileOutputStream(manifestFile), "UTF-8"));
```

Comment:
@wanddy This change should probably be in its own PR so it winds up in its own commit.